frontends/web/src/locales/en/app.json

```diff
@@ -216,9 +216,9 @@
     "noPasswordMatch": "Passwords did not match, please try again.",
     "pairing": {
       "failed": "Unconfirmed pairing. Please replug your BitBox02.",
-      "paired": "You have confirmed on your device that the code matches. If this is correct, you can continue by clicking the button below.",
+      "paired": "Please ensure you verified the pairing code on the device and continue.",
       "title": "Verify pairing code",
-      "unpaired": "A new BitBox02 has been detected. Please verify that the following code matches what is shown on your device. If the code matches, touch underneath the check mark on your BitBox02 and then click the button below to continue."
+      "unpaired": "A new BitBox02 has been detected. Please verify the pairing code matches what is shown on your BitBox02."
```

Comment:
It is not necessarily a new BitBox02, can also be the same one that lost the pairing for many reasons (new compter, paired with too many other wallets, etc).

Reply:
yes you are right, but this is an edge case, most of the time it is a new BitBox. "new" as in new to the app, first time seen, not new unboxed. I think it is better to mention something along this line. Maybe instead of "new" it would be "unseen" or "unpaired" ? @benma  any preference?

Reply:
"An unpaired BitBox02 has been detected..." sounds good to me.

Reply:
updated pr and locize